coralproject/talk
# [Request for feedback] Heroku fix

## What does this PR do?
The Deploy to Heroku button had been removed following a bug report https://github.com/coralproject/talk/issues/1685.

This PR makes a few tweaks to the Talk repo to facilitate one-click deployment with Heroku.

- Omits `package-lock.json` on Heroku to use Yarn instead.

- Silences rewrite env errors, which throws for the Mailgun SMTP password even if one is set through Heroku. Maybe this is due to when the Heroku env is available.

## How do I test this PR?
- Create a new app on Heroku using [this link](https://dashboard.heroku.com/new?template=https%3A%2F%2Fgithub.com%2Fdavisshaver%2Ftalk-heroku&env[TALK_FACEBOOK_APP_ID]=ignore&env[TALK_FACEBOOK_APP_SECRET]=ignore&env[REWRITE_ENV_SILENT]=true).
- The app should build without error.
